CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## v1.4.0 - 2022-01-24
+
+### Added
+
+- #83 - Add support for Python 3.10
+- #59 - Add proper documentation published in Read the doc
+- #70 - Add `add_or_update()` method to DiffSync class that requires a DiffSyncModel to be passed in and will attempt to add or update an existing object
+- #80 - Add support for an existing Diff object to be passed to `sync_to()` & `sync_from()` to prevent another diff from being calculated.
+- #87 - Add new model flags : `SKIP_UNMATCHED_BOTH`, `SKIP_UNMATCHED_SRC` & `SKIP_UNMATCHED_DST` to match the behavior of the global flags
+- #81 - Add a new example based on PeeringDB
+- #53 - Add a new example based on pynautobot and Nautobot REST API
+- #75 - Add support for Structlog v21 in addition to v20.
```

Comment:
Can we put these in numerical order? The current order seems random to me.

Reply:
I tried to bundle the update by topics ... as much as possible
I did it sequentially initially but I found it harder to read ... happy to change

Reply:
Personally I'd prefer sequential order just for sake of consistency. Thanks.